Thai/Activities/Canoeing

```diff
@@ -0,0 +1 @@
+You can canoeing in Southern part of Thailand. Very beautiful beach :)
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.